app/scripts/views/complete_reset_password.js

```diff
@@ -56,10 +56,11 @@ define(function (require, exports, module) {
         });
     },
 
-    afterRender: function () {
+    afterVisible () {
```

Comment:
I changed this from `afterRender` to `afterVisible` because this view re-renders itself, which would cause multiple notifications to be sent.